libs/solid/core/src/lib/components/media-toolbar/media-toolbar.component.ts

```diff
@@ -151,13 +136,29 @@ export class MediaToolbarComponent implements OnInit, OnChanges {
     }
   }
 
+  ngOnInit(): void {
+    this._breakpointObserver
+      .observe(['(max-width: 440px)'])
```

Comment:
Ich glaube, hier müssen wir nochmal dran für den `description` Dialog
(das ist doch hier? Oder ist das in `media-dialog-component`?)

440px ist schon sehr knapp:

![image](https://user-images.githubusercontent.com/13869236/151461210-4bdb8666-f002-4936-a7c5-84e6a668abec.png)

Ich denke, dass ca. 680px ein guter Punkt für den "Flip" des Dialogs nach oben über die Buttons ist. 
Und kannst Du es dann so einrichten, dass der Dialog dann (also < 600px) zentriert und mehr Platz nutzt, z.B. 90%?